### PR TITLE
Adds multiple solution for finding max number in a List with executio…

### PR DESCRIPTION
**Pull request by balag3**
…n time measurement.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 *.class
+*.idea
```
Comment (aron-kovacs) [inline .gitignore]: please add *.iml here :)

File: src/main/java/com/codecool/Service.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package com.codecool;
+
+import java.util.*;
+
+public class Service {
+
+    public static List<Integer> myList = new ArrayList<Integer>(Arrays.asList(55, 2, 4, 45, 65, 6, 41, 63, 5234));
+
+    public static void builtInFindMax() {
```
Comment (aron-kovacs) [inline src/main/java/com/codecool/Service.java]: Please create instance methods instead static ones.

**Comment by balag3** [comment]
By the time I sent the pull request I already worried about the static methods ... :)
The List could stay static?

**Comment by aron-kovacs** [comment]
yep, sure

File: sdt-week-2-collections-balag3.iml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aron-kovacs) [inline sdt-week-2-collections-balag3.iml]: Also if you add *.iml to gitignore, you could delete the already added one :P

**Pull request merged**